src/pages/project/userSettings.tsx

```diff
@@ -0,0 +1,83 @@
+import Header from "@/src/components/layouts/header";
```

Comment:
Right now, i get a 404 error when i hit the name of the user on the bottom left. Can you fix that?